src/main/java/edu/greenblitz/robotName/commands/ReleaseNoteFromArmOnReady.java

```diff
@@ -0,0 +1,47 @@
+package edu.greenblitz.robotName.commands;
+
+import edu.greenblitz.robotName.Field;
+import edu.greenblitz.robotName.commands.arm.roller.ReleaseNoteFromRollerToAmp;
+import edu.greenblitz.robotName.subsystems.arm.elbow.Elbow;
+import edu.greenblitz.robotName.subsystems.arm.elbow.ElbowConstants;
+import edu.greenblitz.robotName.subsystems.arm.wrist.Wrist;
+import edu.greenblitz.robotName.subsystems.arm.wrist.WristConstants;
+import edu.greenblitz.robotName.subsystems.swerve.chassis.SwerveChassis;
+import edu.greenblitz.robotName.utils.AllianceUtilities;
+import edu.greenblitz.robotName.utils.GBCommand;
+import edu.wpi.first.math.geometry.Pose2d;
+
+public class ReleaseNoteFromArmOnReady extends GBCommand {
+
+    private boolean isElbowReady;
+
+    private boolean isWristReady;
+
+    private boolean isRobotInPlace;
+
+    private Pose2d ampScoringPose;
+
+
+    @Override
+    public void initialize() {
+        super.initialize();
+        ampScoringPose = AllianceUtilities.isBlueAlliance() ? Field.ScoringPositions.BLUE_AMP_SCORE_POSITION : Field.ScoringPositions.RED_AMP_SCORE_POSITION;
+    }
+
+    @Override
+    public void execute() {
+        isElbowReady = Elbow.getInstance().isAtAngle(ElbowConstants.PresetPositions.SCORE.ANGLE);
+        isWristReady = Wrist.getInstance().isAtAngle(WristConstants.PresetPositions.SCORE.ANGLE);
+        isRobotInPlace = SwerveChassis.getInstance().isAtPose(ampScoringPose);
```

Comment:
Isatpose has a very low tolerance that I dont think will really work

Reply:
What tolerance should it be?

Reply:
I changed it to 2.5 deg and 0.1 meters